packages/next/build/webpack-config.ts

```diff
@@ -461,6 +461,7 @@ export default async function getBaseWebpackConfig(
             development: dev,
             hasReactRefresh: isMiddleware ? false : hasReactRefresh,
             hasJsxRuntime: true,
+            pageExtensions: config.pageExtensions,
```

Comment:
```suggestion
            nextConfig: config,
```

Let's mirror the `next-swc-loader` options